packages/@o3r/components/schematics/index.it.spec.ts

```diff
@@ -5,6 +5,7 @@
  */
 const o3rEnvironment = globalThis.o3rEnvironment;
 
+import fs from 'node:fs';
```

Comment:
when not necessary you can avoid simplified import:
```suggestion
import * as fs from 'node:fs';
```